uwsgi.h

```diff
@@ -2717,6 +2717,7 @@ struct uwsgi_server {
 #ifdef UWSGI_SSL
 	int ssl_initialized;
 	int ssl_verbose;
+	int ssl_verify_depth;
```

Comment:
:+1: yeah, totally. I just assumed because it was used in the flags that it was part of the CLI and not library that grouping args would make more sense, but on second thought that makes total sense. I even see other SSL flags throughout the struct - sorry for not picking up on that!